### PR TITLE
Fix init pack without filling metadata

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/38435

## Description
Fixed init command when trying to init a pack without filling the metadata file.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/init/initiator.py [hunk 1]
```diff
@@ -238,11 +238,11 @@ def pack_init(self) -> bool:
 
         metadata_path = os.path.join(self.full_output_path, 'pack_metadata.json')
         with open(metadata_path, 'a') as fp:
-            user_response = input("\nWould you like fill pack's metadata file? Y/N ").lower()
+            user_response = str(input("\nWould you like fill pack's metadata file? Y/N "))
```
Comment (roysagi) [inline demisto_sdk/commands/init/initiator.py:241]: why drop the lower?

File: demisto_sdk/commands/init/tests/initiator_test.py [hunk 2]
```diff
@@ -263,6 +263,13 @@ def test_create_metadata_community(self, monkeypatch, initiator):
         }
 
 
+def test_pack_init_without_filling_metadata(monkeypatch, mocker, initiator):
```
Comment (roysagi) [inline demisto_sdk/commands/init/tests/initiator_test.py:266]: please add given when then docstring

File: demisto_sdk/commands/init/initiator.py (hunk 1 above)
Comment (yaakovpraisler) [reply demisto_sdk/commands/init/initiator.py:241]: mistake

File: demisto_sdk/commands/init/tests/initiator_test.py (hunk 2 above)
Comment (yaakovpraisler) [reply demisto_sdk/commands/init/tests/initiator_test.py:266]: done.

File: demisto_sdk/commands/init/tests/initiator_test.py [hunk 3]
```diff
@@ -263,6 +263,21 @@ def test_create_metadata_community(self, monkeypatch, initiator):
...
+    """
+    Given
+        - Pack init inputs.
+    When
+        - Creating new pack without filling the metadata file.
+    Then
+        - Ensure it's not fail.
```
Comment (roysagi) [inline demisto_sdk/commands/init/tests/initiator_test.py:273]:
```suggestion
        - Ensure it does not fail.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,7 @@
 * Added a version differences section to readme in **generate-docs** command.
 * Added the *--docs-format* flag in the **integration-diff** command to get the output in README format.
 * Added the *--input-old-version* and *--skip-breaking-changes* flags in the **generate-docs** command to get the details for the breaking section and to skip the breaking changes section.
+* Fixed an issue where running **init** command without filling the metadata file.
```
Comment (roysagi) [inline CHANGELOG.md:10]: wait until a new version is released and put this in the next RN batch.

**Review by roysagi — APPROVED** [review]

**Pull request merged**